pilot_experiment/3_explore_high_coExpressed_genes.ipynb

```diff
@@ -0,0 +1,2871 @@
WUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUn8yCAzszXM6v7R/g7yRCKztZJ7vMzMzIB333233iFYJ+DEy8zMVsmZZ57JpptuSu/evRk5ciQ33XTTsnEXXXQRI0aMWDbuscceA2DGjBl8/vOfZ+DAgQwYMICjjjpq2TyXXHIJI0aMYIMNNmDPPfdk+vTpAEQExx13HIMGDaJPnz58+MMfZvLkyQDccccdjBw5kt69ezN06FDOPvvsFWLYbLPN6N+/P/vssw+zZs1aNk4Sv/zlL9l8883ZfPPN+da3vsXxxx+/Qvn22Wcfzj333PavOOuUnHiZmdkq2XTTTXnggQeYP38+p556KmPHjmX27Nlcd911nHbaaVxxxRUsWLCAW2+9lQEDBrB06VL23ntvGhoamDZtGjNnzuSggw4C4JZbbuEnP/kJN954I3PnzmXnnXdm9OjRANx1113cf//9PPfcc8yfP59rr72WAQMGAPCVr3yFCy64gDfffJPJkyez2267AXDvvfdy0kknce211zJ79mwaGhqWravRzTffzCOPPMKUKVM49NBDufrqq3nvvfcAmDdvHnfffTdjxowpqzptLefEy8zMVskBBxzAkCFD6NKlCwceeCCbb745EyZM4OKLL+aEE05gu+22QxKbbbYZDQ0NTJgwgVmzZnHWWWfRs2dPunfvzk477QTAb37zG0466SRGjBhB165dOfnkk3niiSeYPn063bp148033+TZZ58lIhgxYgSDBw8GoFu3bkyZMoUFCxawwQYbsO222wIwbtw4Dj/8cLbddlvWW289zjjjDB566CGmTZu2LP6TTjqJ/v3706NHD7bffnv69u3LPffcA8A111zDrrvuykYbbVRupdpay4mXmZmtkiuuuIJtttmGfv360a9fPyZPnsy8efOYMWMGm2666UrTz5gxg4aGBrp2Xfn7XdOnT+eYY45Ztqz+/fsTEcycOZPddtuNo446im9961sMGjSII488kgULFgBwww03cMcdd9DQ0MAuu+zCQw89BMCsWbNoaGhYtvxevXoxYMAAZs6cuWzYsGHDVojh0EMP5aqrrgLgqquu4uCDD171SjLLnHiZmVmbTZ8+na9+9av84he/4LXXXuONN95gq622IiIYNmwYf//731eaZ9iwYbz88stVb2YfNmwYF1xwAW+88cayv7feeosddtgBgKOPPppJkyYxZcoUnnvuOc466ywAtttuO2655RbmzJnDfvvtxxe/+EUAhgwZsuweMYBFixbx2muvMXTo0GXDJK0Qw9ixY7nlllt48sknmTp1Kvvtt9+qV5RZ5sTLzMzabNGiRUhi4MCBAFx66aXLbng/4ogjOPvss5k0aRIRwQsvvMD06dPZfvvtGTx4MCeeeCKLFi1i8eLFPPjggwB8/etf54wzzuCZZ54BYP78+Vx33XUAPProozzyyCMsWbJk2SXKLl268M477zBu3Djmz59Pt27d6NOnD126pNPb6NGjufTSS3niiSd4++23Ofnkk/nYxz7G8OHDa5Zp4403ZrvttuPggw9m//33p0ePHqur+qwT8u94mZmtYTrS72yNHDmS448/no9//ON06dKFQw45hB133BFI93699tprjBkzhpkzZzJ8+HCuvPJKGhoauO222zj66KPZZJNNkMSYMWPYcccd+dznPsfChQs56KCDmD59On379mWPPfbggAMOYMGCBRx33HG8+OKLdO/enT333JPvfe97AFx55ZUcddRRLF26lC222IJx48YBsPvuu3P66aez//778/rrr7PDDjtwzTXXNFuuQw89lIMPPpjzzz9/9VWedUqKiHrHYB3YqFGjYuLEifUOw2xlq/tXRJtSYuYzdepURowYUdr6LLn//vsZO3Ys06dPX+lSZFGt7SNpUkSMWp0x2prJlxrNzMwKlixZwvnnn88RRxzRZNJl1hZOvMzMzLKpU6fSr18/Zs+ezbHHHlvvcGwt5Hu8zMzMshEjRrBo0aJ6h2FrMfd4mZmZmZXEiZeZWQfX+Pga61i8XawtnHiZmXVgPXv2ZObMmbzzzjv4W+gdQ0TwzjvvMHPmTHr27FnvcGwN43u8zMw6sI033ph58+Yxffr0qr/0bvXRtWtX+vbty4YbbljvUGwN48Srk5F0HHAEEMDTwJcjYnF9ozKzWrp06cKgQYMYNGhQvUMxs3bgS42diKShwNHAqIjYClgHOKi+UZmZmXUeTrw6n65AD0ldgfWBWXWOx8zMrNNw4tWJRMRM4GzgZWA2MD8i7qqcTtKRkiZKmjh37tyywzQzM1trOfHqRCRtAOwLvB8YAvSUNLZyuoi4MCJGRcSogQMHlh2mmZnZWsuJV+eyO/BSRMyNiCXAjcAOdY7JzMys03Di1bm8DPy7pPWVnvz6SWBqnWMyMzPrNJx4dSIR8QhwPfAY6ackugAX1jUoMzOzTsS/49XJRMSpwKn1jsPMzKwzco+XmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxImXmZmZWUmceJmZmZmVxIlXJyOpn6TrJT0raaqkj9c7JjMzs86ia70DsNKdD/wxIr4gaV1g/XoHZGZm1lk48epEJPUFPgGrqVARAAAV9ElEQVQcBhAR7wDv1DMmMzOzzsSXGjs4STdK+oyk9thW7wfmApdKelzSxZJ6tsNyzczMrAWceHV8vwLGAM9LOlPSFquwrK7A
```

Comment:
Yeah, this is interesting. Are genes in the same operon always cotranscribed? If so, you'd expect to see them have very correlated expression, or at least I would think so.

Reply:
My understanding is that genes in the same operon share a TF so they should be co-transcribed., which is why I was surprised by this result as well. But perhaps some of the other reasons you mentioned might explain this trend